src/webui-embed.html

```diff
@@ -119,6 +120,10 @@
                     else
                         el.innerHTML = stats[e];
                 }
+                var phash = stats["pool_hashrate"] * 100;
+                percent = phash !== 0 ? phash / stats["network_hashrate"] : 0;
```

Comment:
This looks overly complex. Why not simply:
```
var percent = stats["pool_hashrate"] / stats["network_hashrate"] * 100;
```

Reply:
I didn't like the `0 / something` but I realize, it's the other way around, that's a problem.